packages/edit-post/src/components/manage-blocks-modal/style.scss

```diff
@@ -54,6 +54,7 @@
 	top: 0;
 	padding: $panel-padding 0;
 	background-color: $white;
+	z-index: 1;
```

Comment:
Really minor comment: but we should use our [`_z-index.scss` mixin](https://github.com/WordPress/gutenberg/blob/master/assets/stylesheets/_z-index.scss) for this. Just add a ...  

`".edit-post-manage-blocks-modal__category-title": 1,` 

... entry in that file, and replace this line with: 

`z-index: z-index(".edit-post-manage-blocks-modal__category-title");`